### PR TITLE
feat: add helmraiser & a generated cert-manager lib from the chart

### PR DESCRIPTION
**Pull request by Duologic**
Read the README

Commits can be reviewed individually.

### CODE REVIEW

**Comment by RichiH** [comment]
Sorry, I thought I had submitted this days ago. Seems it was stuck.

File: scripts/helmraiser/helmraiser.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  exit 1
+fi
+
+HELMCHART=$1
+OUTPUT_DIR=$2
+
+set +u
+OVERWRITTEN=false
+if [ "$3" = "overwrite" ] || [ "$4" = "overwrite" ]; then
+    rm -rf $OUTPUT_DIR/templates
```
[the remark this answers is not in the document]
Comment (Duologic) [reply scripts/helmraiser/helmraiser.sh:18]: That would be awesome. Luckily this is regarded a PoC and should be used with care.

How would you solve this? `cd` into it first?

**Comment by RichiH** [comment]
https://github.com/RichiH/vcsh/blob/master/vcsh is painstakingly
escaped with the best POSIX principles and can serve as a template.

The specific question's answer, you have two ways:
rm -rf "$OUTPUT_DIR"/templates
rm -rf "$OUTPUT_DIR/templates"

**Comment by RichiH** [comment]
Ugh, pretend I can English...

**Comment by Duologic** [comment]
Should get replaced by #320 once grafana/tanka#336 is merged and in a release.

**Pull request closed without merging**